src/tools/css/get-css.js

```diff
@@ -64,6 +66,10 @@ const getVariableCSS = (font) => {
 		for (const axis of Object.keys(variation)) {
 			css = css + `${glue}"${axis}" ${variation[axis]}`;
 			glue = ", ";
+			// Poor man's code formatting
+			if (++counter % maxProps === 0) {
+				glue = `,\n        `;
```

Comment:
Why the trailing spaces?

Reply:
They're indentation on the new line. I considered moving this to a variable (`const indent = "        ";`) and concatting that, if that would make it more clear?

Reply:
Ah yes of course, it does make sense. Somehow in my head I read this as the whitespace being before the newline.

Reply:
IMO in general this code could be clearer, but let's no completely rewrite it now because of this small change.

Reply:
Yeah, this is "Old WF code", the kind of code written in those last moments to finally be able to ship it 😅 I hope to address a refactor here once the actual CSS generation gets improved too.